app/views/article_revisions/index.html.haml

```diff
@@ -0,0 +1,17 @@
+.scroll-panel-header
+   %h3.scroll-panel-title Komentarze do numeru
+.scroll-panel-body
+  %table.table.table-hover.table-admin
```

Comment:
Lepiej byłoby wyświetlić to w postaci listy (np. `dl`), ponieważ komentarz może być dość rozbudowany.